infrastructure/helm/keycloak/keycloak-setup.md

```diff
@@ -0,0 +1,52 @@
+# Install Quantumserverless and Configure Keycloak
```

Comment:
I would like to know your opinions here: @akihikokuroda , @IceKhan13 , @psschwei . We already have a `README.md` in `helm\quantumserverless` maybe these steps can go there (taking into account that we are simplify them).

Reply:
The instructions are simple enough to merge in the README now.

Reply:
+1 for main readme